welly/quality.py

```diff
@@ -152,3 +161,210 @@ def spike_locations(curve):
     Return the indicies of the spikes.
     """
     return
+
```

Comment:
I would put the 'orchestrators' at the top of the module because we are probably going to append more qc tests